netutils/config/parser.py

```diff
@@ -246,15 +247,15 @@ def _build_nested_config(self, line):
                 self.indent_level = spaces
 
             if self.is_banner_start(line):
-                line = self._build_banner(line)
+                line = self._build_banner(line)  # type: ignore
                 if line is None or not line[0].isspace():
                     self._current_parents = ()
                     self.indent_level = 0
                     return line
```

Comment:
We should be able to get rid of this `# type: ignore` by using a different variable name than the item of the for loop here:
```
            if self.is_banner_start(line):
                current_line = self._build_banner(line)
                if current_line is None or not current_line[0].isspace():
                    self._current_parents = ()
                    self.indent_level = 0
                    return current_line
```